app/views/services/index.html.erb

```diff
@@ -31,11 +31,10 @@
             <% f.object.errors.each do |error|  %>
               <p class="govuk-error-message"><%= error.message %></p>
             <% end %>
-            <%= f.text_field :service_name, class: "govuk-input" %>
+            <div class="govuk-hint" id="service-name-hint"> <%= t('services.form_name_hint')%></div>
+            <%= f.text_field :service_name, class: "govuk-input", 'aria-describedby' => 'service-name-hint'%>
```

Comment:
I'll admit to not being sure on this one - I'm guessing this syntax works, but I personally would tend to avoid mixing the short and arrow syntax in a hash of options.

I would use either `'aria-describedby': 'service-name-hint'` or you can use a nested hash `aria: { describedby: 'service-name-hint' }`  the nesting will join  the keys together with a `-` in the attibute name in the html.  

The latter version is more useful if you have multiple `aria` attributes or `data` attributes, but it results in a cleaner, more ruby-like syntax.

Not a blocker for approval though